instance_core_maintenance.tf

```diff
@@ -7,7 +7,7 @@ resource "openstack_compute_instance_v2" "maintenance" {
   flavor_name     = "m1.xlarge"
   key_pair        = "cloud2"
   tags            = []
-  security_groups = ["default"]
+  security_groups = ["default", "rsyslog"]
```

Comment:
Is this change intended? Or did it slip from some file you were working on for the maintenance node?

Reply:
Yes, can be a separate PR though. Since it's not a big change I added it here to avoid Terrafrom removing the manually attached `rsyslog` security group.